pymorphy3/units/base.py

```diff
@@ -67,7 +68,10 @@ def _get_param_names(cls):
         """
         if cls.__init__ is object.__init__:
             return []
-        args, varargs, kw, default = inspect.getargspec(cls.__init__)
+        if sys.version_info[0] == 2:
+            args = inspect.getargspec(cls.__init__)[0]
+        else:
```

Comment:
Do we need further support of `python3`? [Python2 is already deprecated](https://www.python.org/doc/sunset-python-2/) and is no longer maintained. I think we need to move to `python3` and forget about `python2`.

Reply:
If you plan to move away from Python 2 support you may need to revisit the whole code and find other legacy parts (and migrate them too). This might be a good idea for another pull request.

Reply:
Ok, I've created an issue for that #11